handbook/engineering/2021_org.md

```diff
@@ -1,46 +1,32 @@
 
+[2021 headcount plan](https://docs.google.com/spreadsheets/d/1UG4aLB5S8yOSOByia2EQgpZkoPxG5eVfqQqgs1f71Vw/edit#gid=2097845084)
+
 - VP Engineering
-  - Director of Engineering: Core Products
-    - [Search Team](search/index.md)
-      - Engineering Manager: Search
-      - Product Manager: Search
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Campaigns Team](campaigns/index.md)
-      - Engineering Manager: Campaigns
-      - Product Manager: Campaigns
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Code Intelligence Team](code-intelligence/index.md)
-      - Engineering Manager: Code Intel
-      - Product Manager: Code Intel
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Extensibility Team](web/index.md)
-      - Engineering Manager: Extensions
-      - Product Manager: Extensions
-      - Frontend Engineers: 4
-    - [Code Host Integrations Team](web/index.md)
-      - Engineering Manager: Code Host Integrations
-      - Product Manager: Code Host Integrations
-      - Frontend Engineers: 4
-  - Director of Engineering: Core Services
-    - [Backend Infrastructure Team](backend-infrastructure/index.md)
-      - Engineering Manager: Backend Infrastructure
-      - Backend Engineers: 4
-      - Dev Ops Engineers: 2
-    - [Distribution Team](distribution/index.md)
-      - Engineering Manager: Distribution
-      - Product Manager: Distribution
-      - Dev Ops Engineers: 4
-      - Backend Engineers: 2
-    - [Security Team](security/index.md)
-      - Engineering Manager: Security
-      - Backend Engineers: 3
-      - Security Engineers: 3
-    - [Web Infrastructure Team](web/index.md)
-      - Engineering Manager: Web Infrastructure
-      - Frontend Engineers: 4
+  - Director of Engineering - Product
+    - [Search Core](search/index.md#search-core) (future split of current Search team)
+    - [Search Growth](search/index.md#search-growth) (future split of current Search team)
+    - [Campaigns](campaigns/index.md)
+    - [Code Intelligence](code-intelligence/index.md)
+    - [Code Insights](web/index.md#code-insights) (future team, currently owned by Web team)
+    - [Extensions and Integrations](web/index.md#extensions-and-integrations) (future team, currently owned by Web team)
+  - Director of Engineering - Infrastructure
+    - [Cloud](cloud/index.md)
+    - [Deployment and Operations Platform](distribution/index.md#deployment-and-operations-platform)
+    - [Developer Platform](distribution/index.md#developer-platform)
+    - [Security](security/index.md)
+    - [Backend Platform](backend-infrastructure/index.md) (future team, currently owned by Cloud team)
+    - [Web Platform](web/index.md#web-infrastructure) (future team, currently owned by Web team)
+
+Every team here is a "full stack" team that has all the skills and roles necessary to achieve the team's mission.
```

Comment:
```suggestion
Every team here is cross-functional, with all the skills and roles necessary to achieve the team's mission.
```

Reply:
FYI, meta-comment: "cross-functional" always to me "across the _horizontal_ organisational structure": backend engineers, frontend engineers, _designers_, tech writers, etc. "full stack" makes me think vertically: from the browser down to the database/deployment.

Reply:
I have the same perception, and since we highlight "skills and roles" in the same sentence, cross-functional makes more sense to me here.